bio/vcf2maf/vcf2maf/wrapper.py

```
@@ -0,0 +1,62 @@
+# Optional input files
+fasta = snakemake.input.get("fasta")
+if fasta:
+    extra += f" --ref-fasta={quote(fasta)}"
+
+chain = snakemake.input.get("chain")
+if chain:
+    extra += f" --remap-chain={quote(chain)}"
+
+# Acquiring external VEP installation
+# if any...
+vep = snakemake.input.get("vep", "")
```

Ruff (0.12.2)

Undefined name `snakemake`

(F821)